cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -112,28 +113,26 @@ private static int _main() throws UnknownHostException {
             matcher.group(1),
             zkTimeoutMs,
             TimeUnit.MILLISECONDS,
-            matcher.group(2)
+            matcher.group(2) + '/' + frameworkName
```

Comment:
Please remove this added sub-directory it's not needed give the user can specify what ever zk node they want as the base.